pkg/util/utils_supported.go

```diff
@@ -99,11 +99,22 @@ func GetRootlessConfigHomeDir() (string, error) {
 }
 
 // GetRootlessPauseProcessPidPath returns the path to the file that holds the pid for
-// the pause process
+// the pause process.
+// DEPRECATED - switch to GetRootlessPauseProcessPidPathGivenDir
 func GetRootlessPauseProcessPidPath() (string, error) {
 	runtimeDir, err := GetRuntimeDir()
```

Comment:
NIT: Should call the new interface from the old one, for completeness.  Fix if forced to repush.
LGTM

Reply:
They're actually different on the `filepath.Join()` bits - the old one manually appends `libpod` to try and find the Libpod tmpdir.

Reply:
I was thinking of:

GetRootlessPauseProcessPidPathGivenDir(filepath.Join(runtimeDir, "libpod"))